undetected_chromedriver/__init__.py

```diff
@@ -279,6 +298,29 @@ def __init__(
         options.add_argument("--remote-debugging-host=%s" % debug_host)
         options.add_argument("--remote-debugging-port=%s" % debug_port)
 
+
+        if proxy:
+            try:
+                if '@' in proxy:
+                    user_password, ip_port = proxy.split("@")
+                    user, password = user_password.split(':')
+                    ip, port = ip_port.split(':') 
+                    proxy_extension = ProxyExtension(ip, int(port), user, password)
+                    options.add_argument(f"--load-extension={proxy_extension.directory}")
+                else:
+                    ip, port = proxy.split(':') 
+                
+                port = int(port)
+                options.add_argument(f"--proxy-server=%s' % '{ip}:{int(port)}")
```

Comment:
The above code has a username password, there is no password here, maybe it should be indented in "else" here?